pareto/strategic_water_management/run_strategic_model.py

```diff
@@ -160,28 +164,30 @@
 
 # create mathematical model
 """Valid values of config arguments for the default parameter in the create_model() call
- objective: [Objectives.cost, Objectives.reuse]
+ objective: [Objectives.cost, Objectives.reuse, Objectives.subsurface_risk]
  pipeline_cost: [PipelineCost.distance_based, PipelineCost.capacity_based]
  pipeline_capacity: [PipelineCapacity.input, PipelineCapacity.calculated]
  hydraulics: [Hydraulics.false, Hydraulics.post_process, Hydraulics.co_optimize, Hydraulics.co_optimize_linearized]
  node_capacity: [True, False]
  water_quality: [WaterQuality.false, WaterQuality.post_process, WaterQuality.discrete]
  removal_efficiency_method: [RemovalEfficiencyMethod.concentration_based, RemovalEfficiencyMethod.load_based]
  infrastructure_timing: [InfrastructureTiming.false, InfrastructureTiming.true]
+ subsurface_risk: [SubsurfaceRisk.false, SubsurfaceRisk.exclude_risky_wells, SubsurfaceRisk.calculate_risk_metrics]
  """
 
 strategic_model = create_model(
     df_sets,
     df_parameters,
     default={
-        "objective": Objectives.cost,
+        "objective": Objectives.subsurface_risk,
```

Comment:
This should run with `cost` objective and not `subsurface_risk`

Reply:
Done.